contracts/ClaimIssuer.sol

```diff
@@ -9,11 +10,11 @@ contract ClaimIssuer is IClaimIssuer, Identity {
     mapping (bytes => bool) public revokedClaims;
     mapping (bytes32 => address) public identityAddresses;
 
-    constructor(address _owner) public {
-        set(_owner);
+    function setClaimIssuer(address _owner) public {
+        postConstructor(_owner);
     }
 
-    function revokeClaim(bytes32 _claimId, address _identity) public override returns(bool) {
+    function revokeClaim(bytes32 _claimId, address _identity) public override delegatedOnly returns(bool) {
```

Comment:
What is `delegatedOnly` for?

Reply:
Modifier from LibraryLock

Reply:
what i don't get with your `delegatedOnly` modifiers is that, imo, they don't prevent at all to call the implementation directly... your `set` functions are setting `initialized` as `true`, therefore functions can be called as they are not blocked by `delegatedOnly`, on the other hand you could tell me that you don't set the implementation, but in this case, as the `set` functions are public, everyone could call the set function and initialize the implementation contract